pkg/webreg/webreg.go

```diff
@@ -1228,6 +1229,138 @@ const imageExampleLiteral = `- as: repo-e2e
           memory: 2Gi
 `
 
+const quotasAndLeasesPage = `<h2 id="title"><a href="#title">How are Cloud Quota and Aggregate Concurrency Limits Handled?</a></h2>
+<p>
+Jobs that interact with an Infrastructure-as-a-Service (IaaS) cloud provider use credentials shared across the broader
```

Comment:
I would reverse how you ordered the info in this passage and start it off with `In order to ensure that` to immediately clarify what this is doing. And generally, keep it shorter, something like `In order to remain within quota limits for cloud resources, we provide a shared locking system that limits the number of concurrent users for a given account`.

I only read this passage because I wanted to write that there is no high-level problem description and wanted to make sure I didn't miss it.